app/views/govuk_publishing_components/components/_layout_super_navigation_header.html.erb

```diff
@@ -65,140 +65,92 @@
       <h2 id="super-navigation-menu-heading" class="govuk-visually-hidden">
         <%= navigation_menu_heading %>
       </h2>
-      <button
-        aria-controls="super-navigation-menu"
-        aria-expanded="true"
-        aria-label="<%= hide_navigation_menu_text %>"
-        class="gem-c-layout-super-navigation-header__navigation-top-toggle-button"
-        data-text-for-hide="<%= hide_navigation_menu_text %>"
-        data-text-for-show="<%= show_navigation_menu_text %>"
-        data-toggle-desktop-group="hidden"
-        data-toggle-mobile-group="top"
-        data-tracking-key="menu"
-        hidden
-        id="super-navigation-menu-toggle"
-        type="button"
-        data-ga4="<%= {
-          "event_name": "select_content",
-          "type": "header menu bar",
-          "text": "Menu",
-          "index": 1,
-          "index_total": navigation_links.length + 2,
-          "section": "Menu"
-        }.to_json %>"
-      >
-        <span class="gem-c-layout-super-navigation-header__navigation-top-toggle-button-inner">
-          Menu
+
+      <%
+        link = t("components.layout_super_navigation_header.navigation_link")
+        unique_id = SecureRandom.hex(4)
```

Comment:
What's the purpose of this unique number appended to the component ID? It would seem to be unnecessary, as there's only one menu and one button, so the original ID without this number would be sufficient?

Reply:
This is here just because in the [preview of the component there are multiple variations](https://components.publishing.service.gov.uk/component-guide//layout_super_navigation_header/preview).